embassy-stm32/src/eth/v2/mod.rs

```diff
@@ -239,6 +187,142 @@ impl<'d, T: Instance, P: PHY> Ethernet<'d, T, P> {
     }
 }
 
+/// Create a new MII Ethernet driver using 14 pins.
+pub fn new_mii<'d, T: Instance, P: PHY, const TX: usize, const RX: usize>(
```

Comment:
please keep these as Ethernet methods, not freestanding functions.

Reply:
Sure, without the const generic parameter, I will move them back.